examples/streamlit/db-connection/README.md

```diff
@@ -0,0 +1,13 @@
+# DB Connection 
+
+This is a sample app for demonstrating DB connections in Streamlit. To run this app ensure you set the environment variable:
+
+```sh 
+export DB_URI=sqlite:///numbers.db
```

Comment:
add a little note here explaining that we're using sqlite for simplicity but that for other databases. and mention the `dialect+driver://username:password@host:port/database` format that applies to other databases

Reply:
[fixed](https://github.com/ploomber/doc/pull/228/commits/7ac4273a4c39acdc5cec17f52bab749b58d64fe5)